authors/apps/authentication/views.py

```diff
@@ -1,13 +1,20 @@
+import os
+import ssl
+from datetime import datetime, timedelta
+
+import jwt
+import sendgrid
+from decouple import config
 from rest_framework import status
 from rest_framework.generics import RetrieveUpdateAPIView
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
+from sendgrid.helpers.mail import *
 
 from .renderers import UserJSONRenderer
-from .serializers import (
-    LoginSerializer, RegistrationSerializer, UserSerializer
-)
+from .serializers import (LoginSerializer, RegistrationSerializer,
```

Comment:
Please undo this change. The previous chunk of code was way more beautiful than your new change.